readme.md

```diff
@@ -2,109 +2,106 @@
 
 ### Overview
 
-You’ve already worked in small groups to accomplish various labs and exercises, but this time **we’re going to challenge you to work on a whole project with a small team.**
-
-Not only will you be asked to **exercise additional creativity** in designing your own project, your instructors will partner you with other classmates to architect, design, and collaboratively build an API of your own design.
-
```

Comment:
Just didn't like the way it sounded. They've already worked on a whole project with a small team. Kept most of the second paragraph, just moved it below the other one.